src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/ScenarioTests/Common.ps1

```diff
@@ -0,0 +1,179 @@
+            {
+                return "2016-05-01-preview"
+            }
+        }
+
+        return $null
+    }
+}
+
+<#
+.SYNOPSIS
+Create a storage account to be used during testing, and return its details
+#>
+function Create-TestStorageAccount($resourceGroup, $location, $storageName)
+{
+    New-AzureRmStorageAccount -ResourceGroupName $resourceGroup -Name $storageName -Location $location -Type 'Standard_LRS' | Out-Null
+    $accessKey = (Get-AzureRmStorageAccountKey -ResourceGroupName $resourceGroup -Name $storageName).Key1;
+    return @{ Name = $storageName; Key = $accessKey }
+}
+
+<#
+.SYNOPSIS
+Cleans the website resource created during testing
+#>
+function Clean-WebService($resourceGroup, $webServiceName)
+{
+    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback) 
+    {
+        try {
+            LogOutput "Removing web service $webServiceName from resource group $rgName"    
+            Remove-AzureRmMlWebService -ResourceGroupName $resourceGroup -Name $webServiceName -Force
+            LogOutput "Web service $webServiceName was removed."
+        }
+        catch {
+            Write-Warning "Caught unexpected exception when cleaning up web service $webServiceName in group $resourceGroup : $($($_.Exception).Message)"
+        }
+    }
+}
+
+<#
+.SYNOPSIS
+Cleans the storage account created during testing
+#>
+function Clean-TestStorageAccount($resourceGroup, $accountName)
+{
+    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback) 
+    {
+        try {
+            LogOutput "Removing storage account $accountName from resource group $rgName"             
+            Remove-AzureRmStorageAccount -ResourceGroupName $resourceGroup -Name $webServiceName
+            LogOutput "Storage account $accountName was removed."
+        }
+        catch {
+            Write-Warning "Caught unexpected exception when cleaning up storage account $accountName in group $resourceGroup : $($($_.Exception).Message)"
+        }
+    }
+}
+
+<#
+.SYNOPSIS
+Cleans the created resource group
+#>
+function Clean-ResourceGroup($resourceGroup)
+{
+    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback)
+    {
+        try {
+            LogOutput "Removing resource group $resourceGroup" 
+            Remove-AzureRmResourceGroup -Name $resourceGroup -Force
+            LogOutput "Resource group $resourceGroup was removed." 
+        }
+        catch {
+            Write-Warning "Caught unexpected exception when cleaning up resource group $resourceGroup : $($($_.Exception).Message)"
+        }
+    }
+}
+
+<#
+.SYNOPSIS
+Writes a timestamped message to stdout
+#>
+function LogOutput($message)
+{
+    $timestamp = Get-Date -UFormat "%Y-%m-%d %H:%M:%S %Z"
+    Write-Output "[$timestamp]: $message"
```

Comment:
Generally speaking, you don't want to write messages to the output stream - use the Verbose stream or the Debug stream for these kinds of messages.